### ISSUE TITLE
bug/<Encoding Bug within partition>

### ISSUE THREAD

**Issue by flow-ryan** [Unstructured-IO/unstructured#465]
**Describe the bug**
I have an html file which I want to process through partition. I explicitly encode the file when writing as UTF-8 but the partition function messes up the encoding. Even when I try to give UTF-8 as the encoding to the partition function it gets messed up. 

**To Reproduce**
```
from unstructured.partition.auto import partition

html_raw = '<h3 class="l_titel">Jahresabschluss zum Geschäftsjahr vom 01.01.2020 bis zum 31.12.2020</h3>'

test_fn = 'test_file.html'

with open(test_fn, 'w', encoding='utf-8') as f:
    f.write(html_raw)
    
elements = partition(filename=test_fn)
print("\n\n".join([str(el) for el in elements]))
```

**Expected behavior**
Print Output: "Jahresabschluss zum Geschäftsjahr vom 01.01.2020 bis zum 31.12.2020"

**Actual Behavior**
Print Output: "Jahresabschluss zum GeschÃ¤ftsjahr vom 01.01.2020 bis zum 31.12.2020"

**Screenshots**
![image](https://user-images.githubusercontent.com/12072586/230979309-b233dc59-3aed-4e22-986b-47b3eb7b772c.png)

**Desktop (please complete the following information):**
 - OS: Mac
 - Python version 3.10.10

**Additional context**

**Comment by MthwRobinson** [comment]
@flow-ryan - Thanks for reporting this. We're tracking it and will pick it up as soon as we can.

**Comment by flow-ryan** [comment]
I have debugged a bit more and I think the issue is that for some reason the characters are double-encoded. (Thanks ChatGPT for the debugging help)

![image](https://user-images.githubusercontent.com/12072586/231098249-6db4d1cd-fecf-4290-b518-7afc09518cc7.png)

When I go into the _read_xml function of the XMLDocument class I see the following line of code
`document_tree = etree.fromstring(content.encode(), self.parser)`

When I remove the additional encode, everything works normal in my notebook. 

Do you think that makes sense? @MthwRobinson

**Comment by MthwRobinson** [comment]
What version of `unstructured` are you on? We actually got rid of that double encode in `0.5.11` to fix the same issue for documents with Chinese characters. If you're on an earlier version I'd try upgrading to `0.5.11` and see if that fixes it.

**Comment by christinestraub** [comment]
@MthwRobinson It worked normally when I tried to give `utf-8` as the encoding to the `partition` function but not with default encoding. I'll be working on getting it to work with the default encoding.

![Screenshot_1](https://github.com/Unstructured-IO/unstructured/assets/9475974/b0cccff7-eade-41a5-9fab-a41cab9ef67b)

![Screenshot_2](https://github.com/Unstructured-IO/unstructured/assets/9475974/1783ff7e-058a-4559-9597-49af6975ec19)

**Comment by MthwRobinson** [comment]
@christinestraub - Got it, thanks for checking that out!

**Comment by MthwRobinson** [comment]
Closed in #894
